%% Template for a preprint Letter or Article for submission
%% to the journal Nature.
%% Written by Peter Czoschke, 26 February 2004
%%
\documentclass{nature}
%% make sure you have the nature.cls and naturemag.bst files where
%% LaTeX can find them
\bibliographystyle{naturemag}
\title{Challenges in LC/MS--based Proteomics}% less than 90 characters
%% Notice placement of commas and superscripts and use of &
%% in the author list
%\author{Aauthor$^{1,2}$, Bauthor$^2$ \& LastAuthor$^2$}
\begin{document}
\maketitle
\begin{enumerate}
    {\bf \item Instrumental Noise in Liquid Chromatography/Mass Spectrometer}\\
    In the nomenclature of MS, electronic (usually due to Analog-to-Digital converters), neutral and chemical noise that takes its source from the procedures undertaken corrupts the total signal in the measurement window, hence creating significant background artifacts in MS and MS/MS spectrum. That can potentially redeem the proteins that occupy smaller percentage in a mixture thus hindering their identification. To tackle with this, (1) establishing a noise model (by exploiting MS and/or MS/MS outputs in response to a blank input), and (2) explicit mathematical treatment of noise, i.e., de-noising of MS and MS/MS spectrum, stand as technical challenges.
    
    {\bf \item Chromatographic Distortions}\\
    Many factors such as presence of contaminants, low signal-to-noise ratio, the integration of fragment ion counts over a certain cycle followed by convolution and smoothing operations leads significant distortion in the chromatograms. The weak and/or two closely located features, therefore, are difficult to detect. To overcome this, a more sophisticated method to produce chromatographic data, i.e., extracted ion chromatograms (XIC) of the transitions of a peptide generated from all corresponding MS2 spectra, is demanded. 
    
    
    {\bf \item Retention Time Alignment}\\
    To relieve the exhaustive search for the targeted peptide over the entire chromatographic space, the retention time of peptides within each gradient is evaluated, thus restricting the search space. To correct for the retention time shift that occurs due to experimental factors (?) between different runs, several strategies, that employ reference peak groups from the assay library, are presented. Those methods, however, usually correct not for the false- and mis-alignments due to the common fragments observed frequently in the chromatogram. The retention time prediction that covers all such possibilities therefore remains as a challenge.
    
    {\bf \item Mass-to-Ion Ratio Inaccuracy}\\
    Experimental precursor and fragment ion m/z values usually differ from that of the assay library. This poses a problem as a class of peptides shares very similar m/z characteristics, hence complicating the identification of a precursor from its fragment ions, yet demanding an internal calibration solution.
    
    {\bf \item Peak Group Identification}\\
    Fragment ion chromatographic profiles (peak groups--features) of peptides are assessed by detecting coeluting peaks. Current methods performs an exhaustive search merely using certain precursors thus discarding substantial amount of features. Instead, feature extraction must be performed as a whole, from highly condensed chromatograms or MS/MS spectra.
   
    {\bf \item Protein Inference}\\
    Assembly of peptides identified in complex mixtures into the list of proteins, commonly referred to as protein inference, is a highly challenging problem in shotgun proteomic data analysis due to common peptide or protein sequences among different protein expressions, i.e., non-injective mapping from peptides to proteins, and existence of degenerate peptides. Current strategies used by targeted approach revolve around searching for certain peptides that uniquely represent distinct proteins, which in turn ensures the existence of those proteins in the mixture. The uniqueness condition for peptides identified in a mixture, however, does not always hold. Therefore, given the reference protein sequence database, determination of proteins from a list of identified peptides --extracted features from mass spectra-- poses a classification problem.
    
    {\bf \item  Incomplete Protein Sequence Database}\\
    Proteogenomics, a recently emerging research field at the interface of proteomics and genomics, enables detection of proteomic variations and identifications of novel peptides utilizing genomics information and transcriptomic nucleotide sequencing data, the so-called protein sequence database building~\cite{Nesvizhskii2014Proteogenomics}. To date, the protein sequence database is incomplete, i.e., records are missing as a whole. Fortunately, high throughput mass spectrometry--based proteomic data analysis can substantially help to improve the completeness of query answers, by searching for the novel and/or modified peptides in the mass spectrum that are not present in reference protein sequence databases. Traditional proteogenomics methods can later integrate these new discoveries to refine gene models in genomics and transciptomics.
    
    However, the mass spectrometry--based proteomic analysis is heavily based on exhaustive search of known entries in the database although the mass spectrum contains entire information about the mixture, i.e., features. Identification of novel peptides or protein sequences in complex mixtures, i.e., feature extraction from the entire spectra, thus remains as a challenge with the current techniques. 
    
    {\bf \item Detection of Post-Translational Modification (PTM)}\\
    Identifications of proteins including the nature and position of any post-translational modifications, and accurate measurement of their dynamic quantitative changes between conditions (e.g., normal vs disease, or control vs. treated samples) are of the essential of goal of proteomics. Recent technology enables this up to some extend. Upon use of more fragment-ions per precursor and more peptides, above mentioned problems will become more complicated to tackle with. More sophisticated methods are required to use more information from protein database in an efficient way.
    
    comprehensive understanding of the changes driving tumour initiation, progression and metastasis
    elucidate the molecular mechanisms driving cancer
    
    {\bf \item Expression and Quantification of Tissue Content}\\
    The proteomic information of substantial patient tissue cohorts coupled with the clinical findings, i.e., percentages of normal, disease cells, changes after drug treatment etc. can enable comprehensive understanding of the moleculer changes driving tumour initiation, progression and metastasis. Additionally, this database can in turn be used to characterize the complex samples without putting clinical effort facilitating to quantify temporal changes such as responsive to drug treatment and metastasis. In the mathematical context, the technical challenge here is to classify the identified protein groups (whose mapping into different classes such as normal and disease present in the current archive), and further to predict the labels of unknown protein groups --novel records for the archive. (dimensionality reduction + recommendation system --I need to be a bit more clear here)

    {\bf \item Speeding-up High Throughput Analysis}\\   
    High scan speed is crucial to automate the analysis of complex mixtures. The typical strategy is to use larger isolation windows, which in turn leads to highly complex, composite fragment ion spectra from multiple precursors~\cite{Gillet2016MassSA}. Therefore, improvements in instrumentation to cycle faster through smaller isolation windows, while still describing the features with sufficient accuracy that facilitates the subsequent learning and generalization steps, will have very high impact for qualitative analysis of proteome content.
    %  higher dynamic range (to detect signals of cofragmenting peptides of vastly different intensit
    
    {\bf \item Protein Variance --Mutation}\\
    Changes of the sequence of a protein
   % {\bf \item Current Software Design}\\
     
\end{enumerate}















%\begin{affiliations}
 %\item Put institutions in this environment and
 %\item separate with \verb|\item| commands.
%\end{affiliations}
% \begin{abstract}
% For Nature, the abstract is really an introductory paragraph set
% in bold type.  This paragraph must be ``fully referenced'' and
% less than 180 words for Letters.  This is the thing that is
% supposed to be aimed at people from other disciplines and is
% arguably the most important part to getting your paper past the
% editors.  End this paragraph with a sentence like ``Here we
% show...'' or something similar.
% \end{abstract}
% Then the body of the main text appears after the intro paragraph.
% Figure environments can be left in place in the document.
% \verb|\includegraphics| commands are ignored since Nature wants
% the figures sent as separate files and the captions are
% automatically moved to the end of the document (they are printed
% out with the \verb|\end{document}| command. However, tables must
% be manually moved to the end of the document, after the addendum.
% Citation of Einstein's paper \cite{Einstein}.
% \begin{figure}
% %%%\includegraphics{something} % this command will be ignored
% \caption{Each figure legend should begin with a brief title for
% the whole figure and continue with a short description of each
% panel and the symbols used. For contributions with methods
% sections, legends should not contain any details of methods, or
% exceed 100 words (fewer than 500 words in total for the whole
% paper). In contributions without methods sections, legends should
% be fewer than 300 words (800 words or fewer in total for the whole
% paper).}
% \end{figure}
% \section*{Another Section}
% Sections can only be used in Articles.  Contributions should be
% organized in the sequence: title, text, methods, references,
% Supplementary Information line (if any), acknowledgements,
% interest declaration, corresponding author line, tables, figure
% legends.
% Spelling must be British English (Oxford English Dictionary)
% In addition, a cover letter needs to be written with the
% following:
% \begin{enumerate}
%  \item A 100 word or less summary indicating on scientific grounds
% why the paper should be considered for a wide-ranging journal like
% \textsl{Nature} instead of a more narrowly focussed journal.
%  \item A 100 word or less summary aimed at a non-scientific audience,
% written at the level of a national newspaper.  It may be used for
% \textsl{Nature}'s press release or other general publicity.
%  \item The cover letter should state clearly what is included as the
% submission, including number of figures, supporting manuscripts
% and any Supplementary Information (specifying number of items and
% format).
%  \item The cover letter should also state the number of
% words of text in the paper; the number of figures and parts of
% figures (for example, 4 figures, comprising 16 separate panels in
% total); a rough estimate of the desired final size of figures in
% terms of number of pages; and a full current postal address,
% telephone and fax numbers, and current e-mail address.
% \end{enumerate}
% See \textsl{Nature}'s website
% (\texttt{http://www.nature.com/nature/submit/gta/index.html}) for
% complete submission guidelines.
% \begin{methods}
% Put methods in here.  If you are going to subsection it, use
% \verb|\subsection| commands.  Methods section should be less than
% 800 words and if it is less than 200 words, it can be incorporated
% into the main text.
% \subsection{Method subsection.}
% Here is a description of a specific method used.  Note that the
% subsection heading ends with a full stop (period) and that the
% command is \verb|\subsection{}| not \verb|\subsection*{}|.
%\end{methods}
%% Put the bibliography here, most people will use BiBTeX in
%% which case the environment below should be replaced with
%% the \bibliography{} command.
% \begin{thebibliography}{1}
% \bibitem{dummy} Articles are restricted to 50 references, Letters
% to 30.
% \bibitem{dummyb} No compound references -- only one source per
% reference.
% \end{thebibliography}
\bibliographystyle{naturemag}
\bibliography{sample}
% %% Here is the endmatter stuff: Supplementary Info, etc.
% %% Use \item's to separate, default label is "Acknowledgements"

% \begin{addendum}
%  \item Put acknowledgements here.
%  \item[Competing Interests] The authors declare that they have no
% competing financial interests.
%  \item[Correspondence] Correspondence and requests for materials
% should be addressed to A.B.C.~(email: myaddress@nowhere.edu).
% \end{addendum}

% %%
% %% TABLES
% %%
% %% If there are any tables, put them here.
% %%

% \begin{table}
% \centering
% \caption{This is a table with scientific results.}
% \medskip
% \begin{tabular}{ccccc}
% \hline
% 1 & 2 & 3 & 4 & 5\\
% \hline
% aaa & bbb & ccc & ddd & eee\\
% aaaa & bbbb & cccc & dddd & eeee\\
% aaaaa & bbbbb & ccccc & ddddd & eeeee\\
% aaaaaa & bbbbbb & cccccc & dddddd & eeeeee\\
% 1.000 & 2.000 & 3.000 & 4.000 & 5.000\\
% \hline
% \end{tabular}
% \end{table}

\end{document}
